spec/user_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+RSpec.describe User, type: :model do
+  subject do
+    User.new(
+      name: 'John Doe',
+      photo: '
+https://picsum.photos/200/300',
+      bio: 'Developer',
+      posts_counter: 5
+    )
+  end
+  before { subject.save }
+  it 'is valid with valid attributes' do
+    expect(subject).to be_valid
+  end
+  it 'is not valid without a name' do
+    subject.name = nil
+    expect(subject).to_not be_valid
+  end
```

Comment:
- [ ] kindly add a spec test file for the post model and test for its validations and methods.
- title should be present
- validates maximum length of title
- validates numericality of likes and comments_counter
- return five recent comments
- update posts counter.
Please kindly add their corresponding unit test.